addons/hr_attendance/models/hr_attendance.py

```diff
@@ -129,17 +129,18 @@ def _compute_overtime_hours(self):
     @api.depends('employee_id', 'check_in', 'check_out')
     def _compute_display_name(self):
         for attendance in self:
+            tz = request.httprequest.cookies.get('tz') if request else None
```

Comment:
Why do it like this rather than take the user's timezone?

Reply:
`format_time` method uses the user's timezone if not provided.
The time on the header and the pill don't match which will create confusion.

That's why I've used server's timezone for that
![image](https://github.com/user-attachments/assets/957edfec-0dc3-429c-af71-a91f360650e5)

Reply:
Ok! Indeed it would make sense.
From what I see in the code, there are occurrences of `request.cookies` but I see no call for `request.httprequest.cookies`, is it different?

Reply:
Yes, we have occurrences of `request.cookies`, but it's from 18.0
In 17.0, there are no cookies in the request object

So I think we should create FP up to 18.0, and from 18.0 to master, we will use 'request.cookies'

Reply:
Found the pr :innocent: 
https://github.com/odoo/odoo/pull/178072/files#diff-5f1b777eb0bc30f3f494ffe3e1522524cd0822f6c94c8644070a6db7c40b9c34R1560